src/main/kotlin/com/openlattice/chronicle/survey/SurveyApi.kt

```diff
@@ -121,6 +123,7 @@ interface SurveyApi {
         @Path(PARTICIPANT_ID) participantId: String,
         @Query(START_DATE) startDateTime: OffsetDateTime,
         @Query(END_DATE) endDateTime: OffsetDateTime,
+        @Query(THRESHOLD) thresholdInSeconds: Int? = 0,
```

Comment:
Why set the default to 0?

Reply:
0 is equivalent to no threshold. It means any thing that is used more than 0 seconds will show up.